test/unit/query-generator/create-database-query.test.ts

```diff
@@ -0,0 +1,61 @@
+import { buildInvalidOptionReceivedError } from '@sequelize/core/_non-semver-use-at-your-own-risk_/utils/check.js';
+import { expectsql, getTestDialect, sequelize } from '../../support';
+
+const dialectName = getTestDialect();
+
+const notSupportedError = new Error(`Databases are not supported in ${dialectName}.`);
+
+describe('QueryGenerator#createDatabaseQuery', () => {
```

Comment:
We had an unit test that combined all options, but it's not here. Is that something that we want to add or do we feel it's redundant?

Reply:
Done